In-house patch to add support for Solaris 11.3 and 12.0.  This patch
has not yet been submitted upstream due to requirements for 3rd party
CI testing.

puppetlabs/xinetd dependency removal will not be accepted by upstream. It is
required to remove a warning from puppet module list.

diff --git a/metadata.json b/metadata.json
index 2cf9c4b..903fa9e 100644
--- a/metadata.json
+++ b/metadata.json
@@ -29,6 +29,14 @@
       "operatingsystemrelease": [
         "14.04"
       ]
+    },
+    {
+      "operatingsystem": "Solaris",
+      "operatingsystemrelease": [
+        "12.0",
+        "11.4",
+        "11.3"
+      ]
     }
   ],
   "description": "Installs and configures OpenStack Swift (Object Storage).",
@@ -37,7 +45,6 @@
     {"name":"openstack/keystone","version_requirement":">=8.2.0 <9.0.0"},
     {"name":"puppetlabs/rsync","version_requirement":">=0.4.0 <1.0.0"},
     {"name":"puppetlabs/stdlib","version_requirement":">=4.9.0 <5.0.0"},
-    {"name":"puppetlabs/xinetd","version_requirement":">=1.5.0 <2.0.0"},
     {"name":"puppetlabs/concat","version_requirement":">=1.2.0 <2.0.0"},
     {"name":"saz/memcached","version_requirement":">=2.8.1 <3.0.0"}
   ]
